### PR TITLE
Adds DLP samples

### PR DESCRIPTION
**Pull request by bshaffer**

### CODE REVIEW

File: dlp/quickstart.php [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+// Set the string to inspect
+$stringToInspect = 'Robert Frost';
+
+// Only return results above a likelihood threshold, 0 for all
+$minLikelihood = likelihood::LIKELIHOOD_UNSPECIFIED;
+
+// Limit the number of findings, 0 for no limit
+$maxFindings = 0;
+
+// Whether to include the matching string
```
Comment (ryanmats) [inline dlp/quickstart.php]: Consider changing to 'Whether to include the matching string in the response"

File: dlp/src/inspect_file.php [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+    $dlp = new DlpServiceClient();
+
+    // The infoTypes of information to match
+    $usMaleNameInfoType = new InfoType();
+    $usMaleNameInfoType->setName('US_MALE_NAME');
+    $usFemaleNameInfoType = new InfoType();
+    $usFemaleNameInfoType->setName('US_FEMALE_NAME');
+    $infoTypes = [$usMaleNameInfoType, $usFemaleNameInfoType];
+
+    // Whether to include the matching string
```
Comment (ryanmats) [inline dlp/src/inspect_file.php]: Consider changing to 'Whether to include the matching string in the response"

File: dlp/src/inspect_string.php [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+    $dlp = new DlpServiceClient();
+
+    // The infoTypes of information to match
+    $usMaleNameInfoType = new InfoType();
+    $usMaleNameInfoType->setName('US_MALE_NAME');
+    $usFemaleNameInfoType = new InfoType();
+    $usFemaleNameInfoType->setName('US_FEMALE_NAME');
+    $infoTypes = [$usMaleNameInfoType, $usFemaleNameInfoType];
+
+    // Whether to include the matching string
```
Comment (ryanmats) [inline dlp/src/inspect_string.php]: Consider changing to 'Whether to include the matching string in the response"

File: dlp/src/redact_string.php [hunk 4]
```diff
@@ -0,0 +1,78 @@
...
+    $dlp = new DlpServiceClient();
+
+    // The infoTypes of information to match
+    $usMaleNameInfoType = new InfoType();
+    $usMaleNameInfoType->setName('US_MALE_NAME');
+    $usFemaleNameInfoType = new InfoType();
+    $usFemaleNameInfoType->setName('US_FEMALE_NAME');
+    $infoTypes = [$usMaleNameInfoType, $usFemaleNameInfoType];
+
+    // Whether to include the matching string
```
Comment (ryanmats) [inline dlp/src/redact_string.php]: Consider changing to 'Whether to include the matching string in the response"

Comment (bshaffer) [inline dlp/src/redact_string.php]: fixed in all cases.

**Pull request merged**